### PR TITLE
refactor(viewer): return label and id in multiple resource selection (DSP-1842 / DSP-1820)

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves DSP-1842 (Expand the "FilteredResources" by label)
and refactor the multiple selection bug DSP-1820 (in detail: <https://dasch.myjetbrains.com/youtrack/issue/DSP-1820#focus=Comments-4-3346.0-0>)

### CODE REVIEW

File: projects/dsp-ui/src/lib/viewer/views/list-view/list-view.service.ts [hunk 1]
```diff
@@ -9,52 +9,78 @@ export class ListViewService {
...
-    selectedResourcesList = [];
+    selectedResourcesList: ShortResInfo[] = [];
     selectedResourceIdxMultiple = [];
 
     constructor() { }
 
-    viewResource(status: CheckboxUpdate, withMultipleSelection: boolean, selectedResourceIdx: number [], resChecks: MatCheckbox[]): FilteredResouces {
+    viewResource(status: CheckboxUpdate, withMultipleSelection: boolean, selectedResourceIdx: number [], resChecks: MatCheckbox[]): FilteredResources {
+
+        // console.log('status', status);
```
Comment (mdelez) [inline projects/dsp-ui/src/lib/viewer/views/list-view/list-view.service.ts:19]: should all these comments be removed?

Comment (kilchenmann) [reply projects/dsp-ui/src/lib/viewer/views/list-view/list-view.service.ts:19]: Oh yes. I have to clean up....

Comment (kilchenmann) [reply projects/dsp-ui/src/lib/viewer/views/list-view/list-view.service.ts]: Done in f8a4369

Comment (mdelez) [reply projects/dsp-ui/src/lib/viewer/views/list-view/list-view.service.ts]: no worries :)

**Review by mdelez — APPROVED** [review]

**Pull request merged**